tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -693,9 +754,27 @@ private static class CommandFileReport {
     private static class CommandRunWorkflows {
         @Parameter(names = "--help", description = "Prints help for run-workflows-through-wes", help = true)
         private boolean help = false;
-        @Parameter(names = "--config-file-path", description = "Path to config file (default ./tooltesterConfig.yml")
+        @Parameter(names = "--config-file-path", description = "Path to config file")
         private String configFilePath = "tooltesterConfig.yml";
 
+        @Parameter(names = "--directory-to-save-results-to", description = "Name of the directory you want to save the run results to")
+        private String resultDirectory = "results";
+
+    }
+
+    @Parameters(separators = "=", commandDescription = "Uploads run results from the `run-workflows-through-wes` command to a specified dockstore site.")
+    private static class CommandUploadRunResults {
+        @Parameter(names = "--help", description = "Prints help for upload-results", help = true)
+        private boolean help = false;
+        @Parameter(names = "--config-file-path", description = "Path to config file")
+        private String configFilePath = "tooltesterConfig.yml";
+
+        @Parameter(names = "--url-to-upload-to", description = "Where the workflow results are being uploaded to (ex. https://qa.dockstore.org/api)", required = true)
+        private String url;
+
+        @Parameter(names = "--location-of-result-files", description = "Location of result files")
+        private String location = "results";
```

Comment:
By default, you want this and line 761 to have the same values, so I suggest creating a static further up so you don't duplicate.

Reply:
Fixed [26b07fb](https://github.com/dockstore/dockstore-support/pull/461/commits/26b07fb4b3390274a176cde167713dd82e5355c7)